builder/bin/main.go

```diff
@@ -0,0 +1,123 @@
+package main
+
+import (
+	"io/ioutil"
+	"os/exec"
+	"strconv"
+	"time"
+
+	"github.com/deis/deis/builder/bindata"
+
+	"github.com/deis/deis/pkg/boot"
+	Log "github.com/deis/deis/pkg/log"
+	. "github.com/deis/deis/pkg/os"
```

Comment:
dot import in Go is not very Go-idiomatic, see [here](https://github.com/golang/go/wiki/CodeReviewComments#import-dot). Maybe name the package `oswrap` and simply import it here?